Bottom: fdee14444e99d6503463cabe41c9264a2807b185
Top:    3b5cff184cf0990b5f9e30912c39f70136a60fe9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 18:27:06 +0000

report.c: Fix missing return value


---

diff --git a/src/report.c b/src/report.c
index be6d1e6..a9c429d 100644
--- a/src/report.c
+++ b/src/report.c
@@ -327,7 +327,7 @@ ReportDialog (int argc, char **argv, int x, int y)
 	  {
 	    LayerTypePtr layer = (LayerTypePtr) ptr1;
 	    __r_dump_tree (layer->polygon_tree->root, 0);
-	    return;
+	    return 0;
 	  }
 #endif
 	Polygon = (PolygonTypePtr) ptr2;
